packages/core/src/tools/utils.ts

```diff
@@ -9,39 +10,6 @@ export const ONE_YEAR = 365 * ONE_DAY
 export const ONE_KIBI_BYTE = 1024
 export const ONE_MEBI_BYTE = 1024 * ONE_KIBI_BYTE
 
-export const enum DOM_EVENT {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏I totally agree on moving it to `/browser` 👍